packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -0,0 +1,367 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sample: Meetings</title>
+    <style>
+      body {
+        background-color: #eee;
+      }
+
+      section {
+        background-color: #fff;
+        padding: 1rem;
+        margin: 1rem;
+      }
+
+      .flex {
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+      }
+
+      video {
+        height: 20vh;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>THIS IS A WORK IN PROGRESS SAMPLE</h1>
+    <section>
+      <h2>Meetings</h2>
+      <p>[[APPLICATION DESCRIPTION]]</p>
+    </section>
+
+    <section>
+      <h2>Authentication</h2>
+      <form id="credentials">
+        <fieldset>
+          <legend>Credentials</legend>
+          <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" />
+          <input id="access-token-save" title="save" type="submit" value="Save" />
+          <label id="access-token-status">[[CREDENTIALS STATUS]]</label>
+        </fieldset>
+      </form>
+
+      <br />
+
+      <form>
+        <fieldset>
+          <legend>Registration</legend>
+          <input id="registration-register" title="register" type="submit" value="Register" />
+          <input id="registration-unregister" title="unregister" type="submit" value="Unregister" />
+          <label id="registration-status">[[REGISTRATION STATUS]]</label>
+        </fieldset>
+      </form>
+    </section>
+
+    <section>
+      <h2>Meetings Management</h2>
```

Comment:
I think we should describe each flow and indirectly explain why the flow is ordered this way

I still get confused by half the stuff within the kitchensink, what's it function and why it's placed there

```suggestion
      <h2>Meetings Management</h2>
      <blockquote>[[MEETING MANAGEMENT DESCRIPTION]]</blockquote>
```

Reply:
You know, I actually had these implemented and then saw that they were consuming too much space. I'll drop these back in.